core/domain/stats_domain.py

```diff
@@ -216,13 +328,14 @@ def validate(self):
                     self.exp_version))
 
         exploration_stats_dict = self.to_dict()
-
+        # Ignore[misc] is used here because mypy does not recognize that key is
```

Comment:
I don't understand this, could you please clarify what you mean by "key is a string"?

Also /cc @vojtechjelinek to double-check that this is OK and that there isn't a way around it.

Reply:
Done (updated comment) -  mypy expects TypedDict key to be string literal, but because the key is being accessed by a variable instead of a string, mypy does not recognize that the key is a string.